velox/exec/Driver.cpp

```diff
@@ -288,7 +288,9 @@ void Driver::init(
     std::vector<std::unique_ptr<Operator>> operators) {
   VELOX_CHECK_NULL(ctx_);
   ctx_ = std::move(ctx);
-  cpuSliceMs_ = ctx_->queryConfig().driverCpuTimeSliceLimitMs();
+  cpuSliceMs_ = task()->executionMode() == Task::ExecutionMode::kSerial
```

Comment:
nit: How about we add method: task::driverCpuSliceMs() for this, and comment the method with PR description? Thanks!